--- mozilla/browser/app/firefox-branding.js~	2006-09-12 18:50:19.000000000 +0300
+++ mozilla/browser/app/firefox-branding.js	2008-10-03 11:14:14.032645886 +0300
@@ -1,5 +1,5 @@
-pref("startup.homepage_override_url","http://www.mozilla.org/projects/%APP%/%VERSION%/whatsnew/");
-pref("startup.homepage_welcome_url","http://www.mozilla.org/projects/%APP%/%VERSION%/firstrun/");
+pref("startup.homepage_override_url","http://www.mozilla.com/%APP%/%VERSION%/releasenotes/");
+pref("startup.homepage_welcome_url","http://www.mozilla.com/%APP%/%VERSION%/firstrun/");
 // URL user can browse to manually if for some reason all update installation
 // attempts fail.
 pref("app.update.url.manual", "http://www.mozilla.org/products/%APP%/");
@@ -8,7 +8,7 @@
 pref("app.update.url.details", "http://www.mozilla.org/projects/%APP%/");
 
 // Release notes URL
-pref("app.releaseNotesURL", "http://www.mozilla.org/projects/%APP%/%VERSION%/releasenotes/");
+pref("app.releaseNotesURL", "http://www.mozilla.com/%APP%/%VERSION%/releasenotes/");
 
 // Search codes belong only in builds with official branding
 pref("browser.search.param.yahoo-fr", "");
