--- mozilla/allmakefiles.sh~	2007-04-03 22:04:34.000000000 +0300
+++ mozilla/allmakefiles.sh	2007-06-04 14:02:26.641218344 +0300
@@ -1561,6 +1561,14 @@
         sql ) MAKEFILES_extensions="$MAKEFILES_extensions
             $MAKEFILES_sql"
             ;;
+
+        tidy ) MAKEFILES_extensions="$MAKEFILES_extensions
+            extensions/tidy/Makefile
+            extensions/tidy/src/Makefile
+            extensions/tidy/test/Makefile
+            extensions/tidy/spvalid/Makefile
+            " ;;
+
         schema-validation ) MAKEFILES_extensions="$MAKEFILES_extensions
             extensions/schema-validation/Makefile
             extensions/schema-validation/public/Makefile
--- mozilla/configure~	2007-06-04 14:32:43.192411445 +0300
+++ mozilla/configure	2007-06-04 17:01:47.504921162 +0300
@@ -13168,7 +13168,7 @@
   MOZ_PHOENIX=1
   MOZ_SAFE_BROWSING=1
   MOZ_APP_VERSION=$FIREFOX_VERSION
-  MOZ_EXTENSIONS_DEFAULT=" cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
+  MOZ_EXTENSIONS_DEFAULT=" tidy cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
   cat >> confdefs.h <<\EOF
 #define MOZ_PHOENIX 1
 EOF
--- mozilla/configure.in~	2007-06-04 14:32:42.892404643 +0300
+++ mozilla/configure.in	2007-06-04 17:02:06.255346400 +0300
@@ -4329,7 +4329,7 @@
   MOZ_PHOENIX=1
   MOZ_SAFE_BROWSING=1
   MOZ_APP_VERSION=$FIREFOX_VERSION
-  MOZ_EXTENSIONS_DEFAULT=" cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
+  MOZ_EXTENSIONS_DEFAULT=" tidy cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
   AC_DEFINE(MOZ_PHOENIX)
   ;;
 
--- mozilla/extensions/tidy/src/Makefile.in~	2007-09-04 23:38:53.000000000 +0300
+++ mozilla/extensions/tidy/src/Makefile.in	2007-09-04 23:49:39.434545403 +0300
@@ -104,7 +104,7 @@
 
 EXTRA_DSO_LDOPTS += \
                 $(NSPR_LIBS) \
-                $(LIB_PREFIX)osp.$(LIB_SUFFIX) \
+                -losp \
                 $(NULL)
 
 include $(topsrcdir)/config/rules.mk
--- mozilla/extensions/tidy/src/Makefile.in~	2007-09-04 23:49:39.000000000 +0300
+++ mozilla/extensions/tidy/src/Makefile.in	2007-09-04 23:50:13.205311926 +0300
@@ -111,6 +111,6 @@
 
 INCLUDES += -I$(srcdir)/../tidy/include 
 INCLUDES += -I$(srcdir)/../tidy/src 
-INCLUDES += -I$(srcdir)/../opensp/include 
+INCLUDES += -I/usr/include/OpenSP
 INCLUDES += -I$(srcdir)/.
 
--- mozilla/extensions/tidy/src/Makefile.in~	2007-09-05 01:41:08.000000000 +0300
+++ mozilla/extensions/tidy/src/Makefile.in	2007-09-05 12:21:13.420348101 +0300
@@ -103,6 +103,7 @@
 endif
 
 EXTRA_DSO_LDOPTS += \
+                $(DIST)/lib/$(LIB_PREFIX)xpcomglue.$(LIB_SUFFIX) $(DIST)/lib/$(LIB_PREFIX)xpcom.so \
                 $(NSPR_LIBS) \
                 -losp \
                 $(NULL)
--- mozilla/extensions/tidy/src/SpValid.cpp~	2007-08-27 10:38:12.000000000 +0300
+++ mozilla/extensions/tidy/src/SpValid.cpp	2007-09-04 23:53:25.049666582 +0300
@@ -9,7 +9,7 @@
 #ifdef WIN32
 #include "../opensp/include/config.h"
 #else
-#include "../opensp/config.h"
+#include <OpenSP/config.h>
 #endif
 
 #include "nsMemory.h"
@@ -19,13 +19,13 @@
 
 #include <stdlib.h>
 
-#include "../opensp/include/MessageEventHandler.h"
-#include "NsgmlsMessages.h"
-#include "../opensp/include/sptchar.h"
+#include <OpenSP/MessageEventHandler.h>
+#include <OpenSP/nsgmls/NsgmlsMessages.h>
+#include <OpenSP/sptchar.h>
 #include "SpValid.h"
-#include "../opensp/include/InternalInputSource.h"
+#include <OpenSP/InternalInputSource.h>
 
-#include "../opensp/include/Parser.h"
+#include <OpenSP/lib/Parser.h>
 
 #ifdef SP_NAMESPACE
 using namespace SP_NAMESPACE;
