--- mozilla/allmakefiles.sh~	2007-04-03 22:04:34.000000000 +0300
+++ mozilla/allmakefiles.sh	2007-06-04 14:02:26.641218344 +0300
@@ -1561,6 +1561,14 @@
         sql ) MAKEFILES_extensions="$MAKEFILES_extensions
             $MAKEFILES_sql"
             ;;
+
+        tidy ) MAKEFILES_extensions="$MAKEFILES_extensions
+            extensions/tidy/Makefile
+            extensions/tidy/src/Makefile
+            extensions/tidy/test/Makefile
+            extensions/tidy/spvalid/Makefile
+            " ;;
+
         schema-validation ) MAKEFILES_extensions="$MAKEFILES_extensions
             extensions/schema-validation/Makefile
             extensions/schema-validation/public/Makefile
--- mozilla/configure~	2007-06-04 14:32:43.192411445 +0300
+++ mozilla/configure	2007-06-04 17:01:47.504921162 +0300
@@ -13168,7 +13168,7 @@
   MOZ_PHOENIX=1
   MOZ_SAFE_BROWSING=1
   MOZ_APP_VERSION=$FIREFOX_VERSION
-  MOZ_EXTENSIONS_DEFAULT=" cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
+  MOZ_EXTENSIONS_DEFAULT=" tidy cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
   cat >> confdefs.h <<\EOF
 #define MOZ_PHOENIX 1
 EOF
--- mozilla/configure.in~	2007-06-04 14:32:42.892404643 +0300
+++ mozilla/configure.in	2007-06-04 17:02:06.255346400 +0300
@@ -4329,7 +4329,7 @@
   MOZ_PHOENIX=1
   MOZ_SAFE_BROWSING=1
   MOZ_APP_VERSION=$FIREFOX_VERSION
-  MOZ_EXTENSIONS_DEFAULT=" cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
+  MOZ_EXTENSIONS_DEFAULT=" tidy cookie xml-rpc xmlextras pref transformiix universalchardet webservices inspector gnomevfs auth permissions reporter spellcheck"
   AC_DEFINE(MOZ_PHOENIX)
   ;;
 
--- mozilla/extensions/tidy/spvalid/Makefile.in	2007-06-04 18:14:33.784020668 +0300
+++ mozilla/extensions/tidy/spvalid/Makefile.in	2007-06-04 19:23:03.277335685 +0300
@@ -49,7 +49,7 @@
 EXTRA_DSO_LDOPTS = \
                 $(DIST)/lib/$(LIB_PREFIX)xpcomglue.$(LIB_SUFFIX) \
                 $(NSPR_LIBS) \
-                $(LIB_PREFIX)osp.$(LIB_SUFFIX) \
+                -losp \
                 $(NULL)
 
 LIBS            = \
@@ -64 +64 @@
-INCLUDES += -I$(srcdir)/../opensp/include 
\ No newline at end of file
+INCLUDES += -I/usr/include/OpenSP
--- mozilla/extensions/tidy/spvalid/SpValid.cpp	2007-06-04 18:16:55.107230927 +0300
+++ mozilla/extensions/tidy/spvalid/SpValid.cpp	2007-06-04 19:30:11.547080063 +0300
@@ -9,7 +9,7 @@
 #ifdef WIN32
 #include "../opensp/include/config.h"
 #else
-#include "../opensp/config.h"
+#include <OpenSP/config.h>
 #endif
 
 #include "nsMemory.h"
@@ -19,13 +19,13 @@
 
 #include <stdlib.h>
 
-#include "MessageEventHandler.h"
-#include "NsgmlsMessages.h"
-#include "sptchar.h"
+#include <OpenSP/MessageEventHandler.h>
+#include <OpenSP/nsgmls/NsgmlsMessages.h>
+#include <OpenSP/sptchar.h>
 #include "SpValid.h"
 #include "InternalInputSource.h"
 
-#include "../lib/Parser.h"
+#include <OpenSP/lib/Parser.h>
 ;
 #ifdef SP_NAMESPACE
 using namespace SP_NAMESPACE;
--- mozilla/extensions/tidy/src/Makefile.in~	2007-06-04 17:18:56.188246299 +0300
+++ mozilla/extensions/tidy/src/Makefile.in	2007-06-04 18:19:42.941042959 +0300
@@ -83,7 +83,7 @@
 EXTRA_DSO_LDOPTS = \
                 $(DIST)/lib/$(LIB_PREFIX)xpcomglue.$(LIB_SUFFIX) \
                 $(NSPR_LIBS) \
-                ../spvalid/$(LIB_PREFIX)osp.$(LIB_SUFFIX) \
+                -losp \
                 $(NULL)
 
 LIBS            = \
--- mozilla/extensions/tidy/spvalid/SpValid.h~	2007-06-04 19:30:14.137138981 +0300
+++ mozilla/extensions/tidy/spvalid/SpValid.h	2007-06-04 19:30:15.877178562 +0300
@@ -5,8 +5,8 @@
 #define Nsgmls_INCLUDED 1
 
 #include "SpResult.h"
-#include "ParserApp.h"
-#include "Boolean.h"
+#include <OpenSP/ParserApp.h>
+#include <OpenSP/Boolean.h>
 
 #ifdef SP_NAMESPACE
 using namespace SP_NAMESPACE;
