diff -ur mozilla-1.9.1/browser/app/profile/firefox.js mozilla-1.9.1.new/browser/app/profile/firefox.js
--- mozilla-1.9.1/browser/app/profile/firefox.js	2009-04-24 03:46:36.000000000 +0200
+++ mozilla-1.9.1.new/browser/app/profile/firefox.js	2009-06-05 15:16:38.961559500 +0200
@@ -92,12 +92,12 @@
 // App-specific update preferences
 
 // Whether or not app updates are enabled
-pref("app.update.enabled", true);
+pref("app.update.enabled", false);
 
 // This preference turns on app.update.mode and allows automatic download and
 // install to take place. We use a separate boolean toggle for this to make
 // the UI easier to construct.
-pref("app.update.auto", true);
+pref("app.update.auto", false);
 
 // Defines how the Application Update Service notifies the user about updates:
 //
@@ -187,7 +187,7 @@
 
 // Whether or not the application should check at startup each time if it 
 // is the default browser.
-pref("browser.shell.checkDefaultBrowser", true);
+pref("browser.shell.checkDefaultBrowser", false);
 
 // 0 = blank, 1 = home (browser.startup.homepage), 2 = last visited page, 3 = resume previous browser session
 // The behavior of option 3 is detailed at: http://wiki.mozilla.org/Session_Restore
@@ -491,7 +491,7 @@
 pref("alerts.totalOpenTime", 4000);
 
 pref("browser.xul.error_pages.enabled", true);
-pref("browser.xul.error_pages.expert_bad_cert", false);
+pref("browser.xul.error_pages.expert_bad_cert", true);
 
 // We want to make sure mail URLs are handled externally...
 pref("network.protocol-handler.external.mailto", true); // for mail
@@ -648,10 +648,10 @@
 
 #ifdef MOZ_SAFE_BROWSING
 // Safe browsing does nothing unless this pref is set
-pref("browser.safebrowsing.enabled", true);
+pref("browser.safebrowsing.enabled", false);
 
 // Prevent loading of pages identified as malware
-pref("browser.safebrowsing.malware.enabled", true);
+pref("browser.safebrowsing.malware.enabled", false);
 
 // Non-enhanced mode (local url lists) URL list to check for updates
 pref("browser.safebrowsing.provider.0.updateURL", "http://safebrowsing.clients.google.com/safebrowsing/downloads?client={moz:client}&appver={moz:version}&pver=2.2");
Tylko w mozilla-1.9.1.new/browser/app/profile: firefox.js~
diff -ur mozilla-1.9.1/browser/base/content/baseMenuOverlay.xul mozilla-1.9.1.new/browser/base/content/baseMenuOverlay.xul
--- mozilla-1.9.1/browser/base/content/baseMenuOverlay.xul	2009-04-24 03:46:36.000000000 +0200
+++ mozilla-1.9.1.new/browser/base/content/baseMenuOverlay.xul	2009-06-05 15:17:27.371040870 +0200
@@ -97,14 +97,6 @@
                   label="&helpReleaseNotes.label;"
                   oncommand="openReleaseNotes()"
                   onclick="checkForMiddleClick(this, event);"/>
-        <menuseparator id="updateSeparator"/>
-#ifdef MOZ_UPDATER
-        <menuitem id="checkForUpdates"
-                  accesskey="&updateCmd.accesskey;"
-                  label="&updateCmd.label;"
-                  class="menuitem-iconic"
-                  oncommand="checkForUpdates();"/>
-#endif
         <menuseparator id="aboutSeparator"/>
         <menuitem id="aboutName"
                   accesskey="&aboutCmd.accesskey;"
Tylko w mozilla-1.9.1.new/browser/base/content: baseMenuOverlay.xul~
diff -ur mozilla-1.9.1/browser/components/preferences/advanced.xul mozilla-1.9.1.new/browser/components/preferences/advanced.xul
--- mozilla-1.9.1/browser/components/preferences/advanced.xul	2009-04-24 03:46:37.000000000 +0200
+++ mozilla-1.9.1.new/browser/components/preferences/advanced.xul	2009-06-05 15:17:55.601991901 +0200
@@ -268,12 +268,6 @@
         <tabpanel id="updatePanel" orient="vertical" align="start">
           <label control="autoUpdateGroup">&autoCheck.label;</label>
           <vbox class="indent" id="autoUpdateGroup" role="group">
-#ifdef MOZ_UPDATER
-            <checkbox id="enableAppUpdate"
-                      label="&enableAppUpdate.label;"
-                      accesskey="&enableAppUpdate.accesskey;"
-                      preference="app.update.enabled"/>
-#endif
             <checkbox id="enableAddonUpdate"
                       label="&enableAddonsUpdate.label;"
                       accesskey="&enableAddonsUpdate.accesskey;"
Tylko w mozilla-1.9.1.new/browser/components/preferences: advanced.xul~
diff -ur mozilla-1.9.1/browser/locales/en-US/chrome/overrides/netError.dtd mozilla-1.9.1.new/browser/locales/en-US/chrome/overrides/netError.dtd
--- mozilla-1.9.1/browser/locales/en-US/chrome/overrides/netError.dtd	2009-04-24 03:46:37.000000000 +0200
+++ mozilla-1.9.1.new/browser/locales/en-US/chrome/overrides/netError.dtd	2009-06-05 15:18:58.260444405 +0200
@@ -175,6 +175,6 @@
 <!ENTITY securityOverride.warningText "
 <p>You should not add an exception if you are using an internet connection that you do not trust completely or if you are not used to seeing a warning for this server.</p>
 
-<xul:button xmlns:xul='http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul' id='getMeOutOfHereButton' label='&securityOverride.getMeOutOfHereButton;'/>
 <xul:button xmlns:xul='http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul' id='exceptionDialogButton' label='&securityOverride.exceptionButtonLabel;'/>
+<xul:button xmlns:xul='http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul' id='getMeOutOfHereButton' label='&securityOverride.getMeOutOfHereButton;'/>
 ">
Tylko w mozilla-1.9.1.new/browser/locales/en-US/chrome/overrides: netError.dtd~
