diff -ur mozilla/browser/locales/en-US/chrome/branding/brand.dtd mozilla.new/browser/locales/en-US/chrome/branding/brand.dtd
--- mozilla/browser/locales/en-US/chrome/branding/brand.dtd	2005-09-06 04:54:21.000000000 +0000
+++ mozilla.new/browser/locales/en-US/chrome/branding/brand.dtd	2006-01-15 18:53:10.231714500 +0000
@@ -1,5 +1,5 @@
-<!ENTITY  brandShortName        "Deer Park">
-<!ENTITY  brandFullName         "Deer Park">            
+<!ENTITY  brandShortName        "Mozilla Firefox">
+<!ENTITY  brandFullName         "Mozilla Firefox">            
 <!ENTITY  vendorShortName       "Mozilla">
 
 <!ENTITY  releaseURL            "http://www.mozilla.org/">
diff -ur mozilla/browser/locales/en-US/chrome/branding/brand.properties mozilla.new/browser/locales/en-US/chrome/branding/brand.properties
--- mozilla/browser/locales/en-US/chrome/branding/brand.properties	2005-09-06 04:54:21.000000000 +0000
+++ mozilla.new/browser/locales/en-US/chrome/branding/brand.properties	2006-01-15 18:53:14.511982000 +0000
@@ -1,3 +1,3 @@
-brandShortName=Deer Park
-brandFullName=Deer Park
+brandShortName=Mozilla Firefox
+brandFullName=Mozilla Firefox
 vendorShortName=Mozilla
