https://bugzilla.mozilla.org/show_bug.cgi?id=322928

Index: gfx/src/gtk/nsFontMetricsPango.cpp
===================================================================
RCS file: /cvsroot/mozilla/gfx/src/gtk/nsFontMetricsPango.cpp,v
retrieving revision 1.16.2.2
diff -d -u -p -r1.16.2.2 nsFontMetricsPango.cpp
--- gfx/src/gtk/nsFontMetricsPango.cpp	16 Sep 2005 06:16:02 -0000	1.16.2.2
+++ gfx/src/gtk/nsFontMetricsPango.cpp	20 Feb 2006 19:15:19 -0000
@@ -981,7 +981,7 @@ nsFontMetricsPango::GetPosition(const PR
 
     // Jump to the end if it's not found.
     if (!found) {
-        if (inx = 0)
+        if (inx == 0)
             retval = 0;
         else if (trailing)
             retval = aLength;
