Index: content/canvas/src/Makefile.in
===================================================================
RCS file: /cvsroot/mozilla/content/canvas/src/Makefile.in,v
retrieving revision 1.4
diff -u -r1.4 Makefile.in
--- content/canvas/src/Makefile.in	11 May 2005 23:49:51 -0000	1.4
+++ content/canvas/src/Makefile.in	22 Aug 2005 19:57:51 -0000
@@ -94,4 +94,6 @@
 
 include $(topsrcdir)/config/rules.mk
 
+CXXFLAGS	+= $(MOZ_CAIRO_CFLAGS)
+
 DEFINES += -D_IMPL_NS_LAYOUT
