diff -ur mozilla-1.9.1/browser/app/firefox-branding.js mozilla-1.9.1.new/browser/app/firefox-branding.js
--- mozilla-1.9.1/browser/app/firefox-branding.js	2009-04-24 03:46:36.000000000 +0200
+++ mozilla-1.9.1.new/browser/app/firefox-branding.js	2009-06-05 15:37:37.664081211 +0200
@@ -1,5 +1,6 @@
-pref("startup.homepage_override_url","http://www.mozilla.org/projects/%APP%/%VERSION%/whatsnew/");
-pref("startup.homepage_welcome_url","http://www.mozilla.org/projects/%APP%/%VERSION%/firstrun/");
+pref("browser.startup.homepage_override.mstone", "ignore");
+pref("startup.homepage_override_url","");
+pref("startup.homepage_welcome_url","");
 // The time interval between checks for a new version (in seconds)
 // nightly=8 hours, official=24 hours
 pref("app.update.interval", 28800);
@@ -11,7 +12,7 @@
 pref("app.update.url.details", "http://www.mozilla.org/projects/%APP%/");
 
 // Release notes URL
-pref("app.releaseNotesURL", "http://www.mozilla.org/projects/%APP%/%VERSION%/releasenotes/");
+pref("app.releaseNotesURL", "http://www.mozilla.com/%LOCALE%/%APP%/%VERSION%/releasenotes/");
 
 // Search codes belong only in builds with official branding
 pref("browser.search.param.yahoo-fr", "");
Tylko w mozilla-1.9.1.new/browser/app: firefox-branding.js~
diff -ur mozilla-1.9.1/browser/base/content/browserconfig.properties mozilla-1.9.1.new/browser/base/content/browserconfig.properties
--- mozilla-1.9.1/browser/base/content/browserconfig.properties	2009-04-24 03:46:37.000000000 +0200
+++ mozilla-1.9.1.new/browser/base/content/browserconfig.properties	2009-06-05 15:38:03.339230244 +0200
@@ -1,2 +1,2 @@
 # Do NOT localize or otherwise change these values
-browser.startup.homepage=http://www.mozilla.org/projects/minefield/
+browser.startup.homepage=http://www.pld-linux.org/
Tylko w mozilla-1.9.1.new/browser/base/content: browserconfig.properties~
diff -ur mozilla-1.9.1/browser/locales/en-US/chrome/branding/brand.dtd mozilla-1.9.1.new/browser/locales/en-US/chrome/branding/brand.dtd
--- mozilla-1.9.1/browser/locales/en-US/chrome/branding/brand.dtd	2009-04-24 03:46:37.000000000 +0200
+++ mozilla-1.9.1.new/browser/locales/en-US/chrome/branding/brand.dtd	2009-06-05 15:39:04.844662546 +0200
@@ -1,4 +1,4 @@
-<!ENTITY  brandShortName        "Minefield">
-<!ENTITY  brandFullName         "Minefield">            
+<!ENTITY  brandShortName        "Firefox Community Edition">
+<!ENTITY  brandFullName         "Firefox Community Edition">
 <!ENTITY  vendorShortName       "Mozilla">
 <!ENTITY  logoCopyright         " ">
Tylko w mozilla-1.9.1.new/browser/locales/en-US/chrome/branding: brand.dtd~
diff -ur mozilla-1.9.1/browser/locales/en-US/chrome/branding/brand.properties mozilla-1.9.1.new/browser/locales/en-US/chrome/branding/brand.properties
--- mozilla-1.9.1/browser/locales/en-US/chrome/branding/brand.properties	2009-04-24 03:46:37.000000000 +0200
+++ mozilla-1.9.1.new/browser/locales/en-US/chrome/branding/brand.properties	2009-06-05 15:39:26.210663187 +0200
@@ -1,3 +1,3 @@
-brandShortName=Minefield
-brandFullName=Minefield
+brandShortName=Firefox Community Edition
+brandFullName=Firefox Community Edition
 vendorShortName=Mozilla
Tylko w mozilla-1.9.1.new/browser/locales/en-US/chrome/branding: brand.properties~
